docs/bigquery_snippets.py

```diff
@@ -461,13 +461,9 @@ def do_something_with(_):
         pass
 
     # [START client_list_jobs]
-    jobs, token = client.list_jobs()   # API request
-    while True:
-        for job in jobs:
-            do_something_with(job)
-        if token is None:
-            break
-        jobs, token = client.list_jobs(page_token=token)  # API request
+    job_iterator = client.list_jobs()   # API request
```

Comment:
1. My bad
1. The API request doesn't happen until the iterator is consumed and there might be multiple requests.